docs/index.md

```diff
@@ -471,108 +485,108 @@ Checks if a given address is a zero address
 
-[index.js:44-44](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L44-L44 "Source code on GitHub")
+Returns **([Buffer](https://nodejs.org/api/buffer.html) \| [Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array))** 
 
-SHA3-256 of an RLP of an empty array (a `Buffer`)
+## MAX_INTEGER
 
-Type: [Buffer](https://nodejs.org/api/buffer.html)
+[index.js:14-14](https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd0cc65885dc8731d1bd2facbba78fb39ac/index.js#L14-L14 "Source code on GitHub")
 
-## SHA3_RLP_ARRAY_S
+the max integer that this VM can handle (a `BN`)
 
-[index.js:38-38](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L38-L38 "Source code on GitHub")
+Type: BN
 
-SHA3-256 of an RLP of an empty array (a `String`)
+## privateToPublic
 
-Type: [String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)
+[index.js:330-334](https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd0cc65885dc8731d1bd2facbba78fb39ac/index.js#L330-L334 "Source code on GitHub")
 
-## SHA3_RLP_S
+Returns the ethereum public key of a given private key
 
-[index.js:50-50](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L50-L50 "Source code on GitHub")
+**Parameters**
 
-SHA3-256 hash of the RLP of null  (a `String`)
+-   `privateKey` **[Buffer](https://nodejs.org/api/buffer.html)** A private key must be 256 bits wide
 
-Type: [String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)
+Returns **[Buffer](https://nodejs.org/api/buffer.html)** 
 
 ## TWO_POW256
 
-[index.js:20-20](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L20-L20 "Source code on GitHub")
+[index.js:20-20](https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd0cc65885dc8731d1bd2facbba78fb39ac/index.js#L20-L20 "Source code on GitHub")
 
 2^256 (a `BN`)
 
 Type: BN
 
 ## zeroAddress
 
-[index.js:91-95](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L91-L95 "Source code on GitHub")
+[index.js:97-101](https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd0cc65885dc8731d1bd2facbba78fb39ac/index.js#L97-L101 "Source code on GitHub")
 
 Returns a zero address
 
 Returns **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** 
 
 ## zeros
 
-[index.js:82-84](https://github.com/ethereumjs/ethereumjs-util/blob/0bc0616c2de0de0723f87c3b64923bb9f962a8e1/index.js#L82-L84 "Source code on GitHub")
+[index.js:88-90](https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd0cc65885dc8731d1bd2facbba78fb39ac/index.js#L88-L90 "Source code on GitHub")
```

Comment:
Had a rough look over the documentation and did a short markdown preview, looks good.